scripts/check-env.sh

```diff
@@ -3,16 +3,85 @@
 # Check the environment for dependencies and configuration
 #
 
+# check appropriate package installer to recommend corresponding packages
+RPM_CMD=$(command -v rpm)
+DPKG_CMD=$(command -v dpkg)
+
 # count errors, run script to the end before exiting
 ERRORS=0
 fail() {
 	echo "${*}" > /dev/stderr
 	ERRORS=$((ERRORS+1))
 }
 
+ceph_version() {
+	TOOLBOX_POD=$(kubectl -n rook-ceph get pods -l app=rook-ceph-tools -o jsonpath='{.items[0].metadata.name}')
```

Comment:
This is not possible, we don't have/need kubernetes cluster or ceph cluster during development

Reply:
What I meant in yesterday discuss was , as you are checking the packages are installed, it's also good to check the version of packages installed

Reply:
So do we have to skip ceph version check then?

Reply:
yeah we can skip it now, we can have an enhancement in follow up patch, maybe need to see how to do it in golang

Reply:
Okay. Thanks to clear the misunderstanding. I'll make the changes.